libs/src/api/file.js

```diff
@@ -68,12 +68,21 @@ class File extends Base {
 
     const gateways = creatorNodeGateways.concat(publicGateways)
 
-    return retry(async () => {
+    return retry(async (bail) => {
       try {
-        const { response } = await raceRequests(urls, callback, {
+        const { response, errored } = await raceRequests(urls, callback, {
           method: 'get',
           responseType
         }, /* timeout */ null)
+
+        const allUnauthorized =
+          response === undefined &&
```

Comment:
looking at `raceRequests` it looks like `response` should actually be `null` instead of `undefined`? could it be because of this typo https://github.com/AudiusProject/audius-protocol/blob/master/libs/src/utils/network.ts#L249 ("respone" instead of "response"

or could just do `!response` to mirror line 86 and 101 below

Reply:
oh jeeeeez

Reply:
😅

Reply:
combined with latter check & fixed the typo. much better now.

Reply:
🙌